libs/template/helpers.go

```diff
@@ -17,4 +18,8 @@ var helperFuncs = template.FuncMap{
 	"fail": func(format string, args ...any) (any, error) {
 		return nil, ErrFail{fmt.Sprintf(format, args...)}
 	},
+	// Alias for https://pkg.go.dev/regexp#Compile. Allows usage of all methods of regexp.Regexp
+	"regexpCompile": func(expr string) (*regexp.Regexp, error) {
```

Comment:
Same as for `urlParse`, should we call this just `regexp`?

Reply:
done